### ISSUE TITLE
[Feature Request] Add Color-Blind mode to Windows OS (Colored cmd output)

### ISSUE THREAD

**Issue by j-moriarti** [IngoZenz/personaldnsfilter#52]
Hi! I'm new here (and GitHub!), but I've been using your Amazing app both in Android and Windows.
My request is that, can you add a feature (similar to #40) for windows?
I was curious about this, so with my little knowledge, tried to find a way to implement this feature with trial and error, and somehow got succeeded :smile: (and built a test forked version here : [https://github.com/j-moriarti/personaldnsfilter](https://github.com/j-moriarti/personaldnsfilter). But it is not neat in code and also I didn't know how to pull request :) , so I just came here to request this feature and say Thank you :+1:

**Comment by IngoZenz** [comment]
Hi j-moriarti,
Happy to see your interest on my tiny project :-)
I suggest that you add your changes to DNSFilterProxy.java within the StandAloneLogger Implementation. That's the Logging Impl for the  non-android version.
I fear with the changes you proposed, the logging  within the android version got broken ...
Maybe there is also the possibility to implement it without importing the whole ansi console lib in order to keep it small and without additional dependencies?
 I am curious :-)

**Comment by j-moriarti** [comment]
You are absolutely right! it was just my first attempt at coding in java 😊 and because of being highly dependent to your great project, I tried to edit it. but I don't know much about java coding, so I may have mistakes in coding. I was just requesting from you if it is possible to make a look at it or maybe you could find a better and smaller alternative to that ansi console lib. I could then remove my messy edits 😅

**Comment by IngoZenz** [comment]
Yes - I will take it to the list ;-)

**Comment by j-moriarti** [comment]
Hi again!
I played with code and edited and fixed some of your mentioned issues.
these files are the edited sources of the pDNSf project with the output for android and PC.
[personaldnsfilter-master.zip](https://github.com/IngoZenz/personaldnsfilter/files/4674321/personaldnsfilter-master.zip)
[personaldnsfilter-master-DNSFilter-1.50.40.0-mr2.zip](https://github.com/IngoZenz/personaldnsfilter/files/4674322/personaldnsfilter-master-DNSFilter-1.50.40.0-mr2.zip)
